tests/test_release_cycles.py

```diff
@@ -42,17 +42,13 @@ def test_successful_release_cycle(release_name):
     Expects this release cycle to be successful. 
     """
     ensure_shell_script_integrity()
-    reseed()
+    clear_tables()
```

Comment:
can we not use the `client.put("/clear")` here like in other tests? if yes, we should leverage this and get rid of the `dao/clear_tables_reseed.py` module completely